### PR TITLE
Drop links from TriviaQA

### PR DESCRIPTION
**Pull request by yeganehkordi**
- Dropping the links from TriviaQA tasks
- Update the test all to detect URLs in the tasks

### CODE REVIEW

File: src/test_all.py [hunk 1]
```diff
@@ -211,7 +211,16 @@ def natural_keys(text):
...
-
+                                        
+            # Make sure there are no link in instances
+            url_reg  = r'http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+            instances = data['Instances']    
+            for instance in instances:
+                ck_url = re.findall(url_reg, instance['input'])
+                if ck_url:
+                    print(f'⚠️ WARNING: Looks like there is a link in the input: {ck_url}')
+                    break
```
Comment (danyaljj) [inline src/test_all.py:222]: should this raise an exception (rather than `break`)?

**Comment by danyaljj** [comment]
Thanks! Left a question above.

File: src/test_all.py (hunk 1 above)
Comment (yeganehkordi) [reply src/test_all.py:222]: We have many tasks with links in the input and we need to change some of their definitions. After updating them, I'll change this scope to raise an exception.

Comment (danyaljj) [reply src/test_all.py:222]: I see. Let's remember to address these definitely in near future.

**Pull request merged**